prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html

```diff
@@ -0,0 +1,128 @@
+<app-page>
```

Comment:
Added new functionality to app-page, you can use the following attributes and define a onBack to have the back arrow by the tittle.
[showToolbar]="true"
(back)="onBack()"

Reply:
Also did read the thing above
haha

Reply:
sweet

Reply:
🛥️

Reply:
I know talk real well